controllers/space/space_controller_test.go

```diff
@@ -833,6 +834,77 @@ func TestUpdateSpaceTier(t *testing.T) {
 		})
 	})
 
+	t.Run("tier update for 3 spaces (same tier but updated references)", func(t *testing.T) {
+		// get an older basic tier (with outdated references)
+		olderBasicTier := tiertest.BasicTier(t, tiertest.PreviousBasicTemplates)
+		// given 3 spaces set to the older version of the tier
+		s1 := spacetest.NewSpace("oddity1",
```

Comment:
could you please update the comments - these in only one space - no three of them.
could you please also change the name of the var so it's no just `s1`, but something like `space`?

Reply:
not `space`, it will conflict with the package name